src/index.html

```diff
@@ -22,6 +22,486 @@
+          </div>
+
+          <div class="card__price">
+            <span class="card__price-title">Price:</span>
+
+            <span class="card__price-value">$2,199</span>
+          </div>
+
+          <a
+            href="#"
+            class="card__button"
+            data-qa="hover"
+          >
+            Buy
+          </a>
+        </section>
+      </div>
+
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <img
+          class="card__img"
+          src="./images/imac.jpeg"
+          alt="Product"
+        />
+
+        <section class="card__info">
+          <div class="card__title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </div>
+
+          <span class="card__code">Product code: 195434</span>
+
+          <div class="card__reviews">
+            <div class="stars stars--rating-4">
+              <div class="stars__star stars__star--active"></div>
+
+              <div class="stars__star stars__star--active"></div>
+
+              <div class="stars__star stars__star--active"></div>
+
+              <div class="stars__star stars__star--active"></div>
+
+              <div class="stars__star"></div>
+            </div>
+            <div class="card__reviews-value">Reviews: 5</div>
+          </div>
+
+          <div class="card__price">
+            <span class="card__price-title">Price:</span>
+
+            <span class="card__price-value">$2,199</span>
+          </div>
+
+          <a
+            href="#"
+            class="card__button"
+            data-qa="hover"
+          >
+            Buy
+          </a>
+        </section>
+      </div>
+
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <img
+          class="card__img"
+          src="./images/imac.jpeg"
+          alt="Product"
+        />
+
+        <section class="card__info">
+          <div class="card__title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </div>
+
+          <span class="card__code">Product code: 195434</span>
+
+          <div class="card__reviews">
+            <div class="stars stars--rating-4">
+              <div class="stars__star stars__star--active"></div>
+
+              <div class="stars__star stars__star--active"></div>
+
+              <div class="stars__star stars__star--active"></div>
+
+              <div class="stars__star stars__star--active"></div>
+
+              <div class="stars__star"></div>
+            </div>
+            <div class="card__reviews-value">Reviews: 5</div>
+          </div>
+
+          <div class="card__price">
+            <span class="card__price-title">Price:</span>
+
+            <span class="card__price-value">$2,199</span>
+          </div>
+
+          <a
+            href="#"
+            class="card__button"
+            data-qa="hover"
+          >
+            Buy
+          </a>
+        </section>
+      </div>
```

Comment:
It seems like you have multiple `card` blocks with the same content. Instead of duplicating the code, consider using a loop in your template engine or JavaScript to generate these cards if the content is dynamic or they are placeholders for future dynamic content.